library/src/main/cpp/java_wrapper.cpp

```diff
@@ -107,8 +111,54 @@ Java_tachiyomi_decoder_ImageDecoder_nativeDecode(
     return nullptr;
   }
 
+  cmsHPROFILE target_profile = nullptr;
+  if (apply_cms) {
+    if (icm_stream != NULL) {
```

Comment:
Nit: I prefer to use either `nullptr` or `!icm_stream` since `NULL` is more of a C thing.